blob/service.go

```diff
@@ -26,22 +25,29 @@ var (
 	log = logging.Logger("blob")
 )
 
+// Submitter is an interface that allows submitting blobs to the celestia-core. It is used to
+// avoid a circular dependency between the blob and the state package, since the state package needs
+// the blob.Blob type for this signature.
+type Submitter interface {
+	SubmitPayForBlob(ctx context.Context, fee math.Int, gasLim uint64, blobs []*Blob) (*types.TxResponse, error)
```

Comment:
And break state module? noooo thank you

Reply:
I am against the name change, I think it increases ambiguity between state and blob module methods - if anything, I would change it to SubmitPayForBlobTx. But against changing it in general rn because I don't think its worth any upside

Reply:
Agree that there is no real upside. It's more like my indignation about PayForBlobTx naming. It's like PayForTransferTx, which does not make sense.